TestAssets/TestProjects/PortableApp/project.json

```diff
@@ -0,0 +1,18 @@
+{
+ 
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {},
+  "frameworks": {
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.0.0-rc3-*"
+        },
+        "Newtonsoft.Json": "9.0.1-beta1"
```

Comment:
It is necessary. Portable App build/publish behavior is different between the netcore.app package and  external packages. Introducing this actually brought in some failures @schellap and I had to work through to determine if they were valid scenarios. For example, moving the deps.json from a built app without passing --additionalprobingpath will cause Newtonsoft.Json not to be found. There are some failure scenarios in the tests for htis.

---
In reply to: [66346538](https://github.com/dotnet/core-setup/pull/97#discussion_r66346538) [](ancestors = 66346538)